src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplyForBenefits.unit.spec.jsx

```diff
@@ -32,6 +32,69 @@ function oneYearFromNow() {
   return Date.now() + oneYearInMS;
 }
 
+function loadingSpinnerIsHidden(view) {
```

Comment:
I may have gone a little overboard with the test refactoring. I was trying to do two things:

1. Stop throwing around the same strings/regexes in multiple places
2. But mostly make the tests themselves easier to read